Patches/HLX ReGrowth - Extinct Animals Pack/Extinct_Mammals.xml

```diff
@@ -0,0 +1,237 @@
+								<li>Blunt</li>
+								<li>Poke</li>
+							</capacities>
+							<power>10</power>
+							<cooldownTime>2</cooldownTime>
+							<linkedBodyPartsGroup>FrontRightLeg</linkedBodyPartsGroup>
+							<armorPenetrationBlunt>1.125</armorPenetrationBlunt>
+						</li>
+						<li Class="CombatExtended.ToolCE">
+                						<label>horn</label>
+							<capacities>
+								<li>Scratch</li><li>Blunt</li>
+							</capacities>
+							<power>16</power>
+							<cooldownTime>2</cooldownTime>
+							<linkedBodyPartsGroup>Teeth</linkedBodyPartsGroup>
+							<armorPenetrationBlunt>1.5</armorPenetrationBlunt>
+							<armorPenetrationSharp>0.35</armorPenetrationSharp>
+						</li>
+						<li Class="CombatExtended.ToolCE">
+							<capacities>
+								<li>BiteBlunt</li>
+							</capacities>
+							<power>7</power>
+							<cooldownTime>2</cooldownTime>
+							<linkedBodyPartsGroup>Teeth</linkedBodyPartsGroup>
+							<armorPenetrationBlunt>0.700</armorPenetrationBlunt>
+							<chanceFactor>0.5</chanceFactor>
+						</li>
+						<li Class="CombatExtended.ToolCE">
+							<label>head</label>
+							<capacities>
+								<li>Blunt</li>
+							</capacities>
+							<power>16</power>
+							<cooldownTime>2.2</cooldownTime>
+							<linkedBodyPartsGroup>HeadAttackTool</linkedBodyPartsGroup>
+							<armorPenetrationBlunt>3.125</armorPenetrationBlunt>
+							<ensureLinkedBodyPartsGroupAlwaysUsable>true</ensureLinkedBodyPartsGroupAlwaysUsable>
+						</li>
+					</tools>
+				</value>
+			</li>
+
+        <li Class="PatchOperationReplace">
+		<xpath>Defs/PawnKindDef[defName="RG-EAP-Protoceratidae"]/combatPower</xpath>
+		<value>
+			<combatPower>50</combatPower>
+		</value>
+        </li>
+
+        <!-- === Wooly Rhino === -->
+						
+			<li Class="PatchOperationAdd">
+				<xpath>/Defs/ThingDef[defName="RG-EAP_WoollyRhino"]/statBases</xpath>
+				<value>
+				<MeleeDodgeChance>0.06</MeleeDodgeChance>
+				<MeleeCritChance>0.41</MeleeCritChance>
+				<MeleeParryChance>0.3</MeleeParryChance>
+				<ArmorRating_Blunt>3</ArmorRating_Blunt>
+				<ArmorRating_Sharp>0.4</ArmorRating_Sharp>
+				</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>/Defs/ThingDef[defName="RG-EAP_WoollyRhino"]/statBases/MoveSpeed</xpath>
+				<value>
+					<MoveSpeed>4.8</MoveSpeed>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>/Defs/ThingDef[defName="RG-EAP_WoollyRhino"]/tools</xpath>
+				<value>
+					<tools>
+						<li Class="CombatExtended.ToolCE">
+							<label>horn (Blunt)</label>
+							<capacities><li>Blunt</li></capacities>
+							<power>30</power>
+							<cooldownTime>2.33</cooldownTime>
+							<linkedBodyPartsGroup>HornAttackTool</linkedBodyPartsGroup>
+							<armorPenetrationBlunt>22</armorPenetrationBlunt>
+						</li>
+						<li Class="CombatExtended.ToolCE">
+							<label>horn</label>
+							<capacities><li>Stab</li><li>Scratch</li></capacities>
+							<power>36</power>

```

Comment:
I'd scale this down and remove the `Scratch` capability, and may consider swapping the Stab for a Poke to match the vanilla rhino. The vanilla rhino does 24 damage, and while the wooly rhino is slightly larger from what I can tell, it's not much larger. I'd set both horn attacks to 28 - 32 range.